ddp-workspace/projects/ddp-dsm-ui/src/app/scan/scan.component.ts

```diff
@@ -156,6 +157,7 @@ export class ScanComponent implements OnInit {
       if (this.scanPairs.length < 1) {
         const newScanPair = new ScanPairComponent();
         this.scanPairs.push(newScanPair);
+        this.scanPairsValue.push(new ScanPair('', ''));
```

Comment:
I think it would be better if empty strings were default. like so:

```
export class ScanPair {
  public error: string;

  constructor(public leftValue: string = '',  public rightValue: string = '') {}
}

```

Then you would just instantiate this class like so: `new ScanPair()`

Reply:
Done diddly did 👍